src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2328,7 +2328,17 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="QuickAccent_ToolbarPosition_Right.Content" xml:space="preserve">
     <value>Right</value>
   </data>
-  <data name="LearnMore_TextExtractor.Text" xml:space="preserve">
+  <data name="QuickAccent_Behavior.Header" xml:space="preserve">
+    <value>Behavior</value>
+  </data>
+  <data name="QuickAccent_InputTimeMs.Header" xml:space="preserve">
+    <value>Input delay (ms)</value>
+    <comment>ms = milliseconds</comment>
+  </data>
+  <data name="QuickAccent_InputTimeMs.Description" xml:space="preserve">
+    <value>Hold the key for this long to make the accent menu appear (ms)</value>
```

Comment:
Maybe some rephrasing here. It's not actually about holding the key but the response time of quick accent, as you can just press the key and the toolbar will appear after 'Input time' if you don't press anything else in the meantime.

Reply:
Not sure what a better phrasing would be here, to be honest.